src/lib.rs

```diff
@@ -70,35 +70,59 @@
 //!
 //! ```{rust}
 //! use semver::Version;
+//! use std::error::Error;
 //!
-//! let mut bugfix_release = Version::parse("1.0.0").unwrap();
+//! # fn try_increment_patch() -> Result<(), Box<Error>> {
+//! let mut bugfix_release = Version::parse("1.0.0")?;
 //! bugfix_release.increment_patch();
 //!
 //! assert_eq!(Ok(bugfix_release), Version::parse("1.0.1"));
+//!
```

Comment:
Let's remove these extra newlines before the hidden lines. Otherwise the docs will have blank lines at the end of the code blocks.